test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -675,12 +679,18 @@ proc ::AlgorandGoal::GetHighestFundedAccountForWallet { WALLET_NAME NODE_DIR } {
         set ACCOUNT_ADDRESS "NONE"
         expect {
             timeout { ::AlgorandGoal::Abort "Timed out finding primary wallet: $WALLET_NAME"  }
+            -re {\[n/a\]\t([a-zA-Z0-9]+)\t([a-zA-Z0-9]+)\t\[n/a\]} {
+                    set CURRENT_ACCOUNT_ADDRESS $expect_out(2,string);
+                    if { $ACCOUNT_BALANCE == 0 } then { set ACCOUNT_ADDRESS $CURRENT_ACCOUNT_ADDRESS };
+                    exp_continue;
+                }
```

Comment:
This adds the support to allow us to get the account even when the algod isn't running.